### PR TITLE
Update to latest nukkitx

### PR DESCRIPTION
**Pull request by boy0001**

### CODE REVIEW

File: src/main/java/nl/rutgerkok/pokkit/player/PokkitPlayer.java [hunk 1]
```diff
@@ -1039,7 +1037,7 @@ public void sendTitle(String title, String subtitle, int fadeIn, int stay, int f
 
 	@Override
 	public void setAllowFlight(boolean value) {
-		nukkit.getAdventureSettings().setCanFly(value);
+		nukkit.getAdventureSettings().set(AdventureSettings.Type.ALLOW_FLIGHT, true);
```
Comment (rutgerkok) [inline src/main/java/nl/rutgerkok/pokkit/player/PokkitPlayer.java]: This should be set to `value` I think, not to `true`

File: src/main/java/nl/rutgerkok/pokkit/player/PokkitPlayer.java [hunk 2]
```diff
@@ -1100,7 +1098,7 @@ public void setFireTicks(int fireTicks) {
 
 	@Override
 	public void setFlying(boolean flying) {
-		nukkit.getAdventureSettings().setFlying(flying);
+		nukkit.getAdventureSettings().set(AdventureSettings.Type.FLYING, true);
```
Comment (rutgerkok) [inline src/main/java/nl/rutgerkok/pokkit/player/PokkitPlayer.java]: Same here.

**Comment by rutgerkok** [comment]
Thank you! I found two places where I think there's a bug, could you take a look at them?

**Comment by twixi** [comment]
its work?

**Comment by rutgerkok** [comment]
Merged!

**Pull request merged**